src/calendar-list/index.tsx

```diff
@@ -250,11 +254,21 @@ const CalendarList = (props: CalendarListProps & ContextProp, ref: any) => {
     );
   }, [horizontal, calendarStyle, calendarWidth, testID, getMarkedDatesForItem, isDateInRange, calendarProps]);
 
+  const onPressArrowLeft = useCallback((method, month) => {
+    headerProps.onPressArrowLeft?.(method, month);
+  }, [headerProps.onPressArrowLeft]);
+
+  const onPressArrowRight = useCallback((method, month) => {
+    headerProps.onPressArrowRight?.(method, month);
+  }, [headerProps.onPressArrowRight]);
+
   const renderStaticHeader = () => {
     if (staticHeader && horizontal) {
       return (
         <CalendarHeader
           {...headerProps}
+          onPressArrowLeft={headerProps.onPressArrowLeft ? onPressArrowLeft : undefined}
```

Comment:
Why do we need to implement these callbacks? The CalendarHeader is invoking them by default..

Reply:
you are right, they were part of a previous attempt to fix and are obsolete now